index.html

```diff
@@ -39,8 +39,205 @@ <h1 class="hero-title" >I’m Mariane <br>
         </div>
     </section>
```

Comment:
Add `padding-left: 12px` to the `LET’S CONNECT` to achieve the design suggested by the previous reviewer.
![image](https://user-images.githubusercontent.com/61361037/219704003-8a459bca-a549-4cf4-81e9-f83c2db73c89.png)